### PR TITLE
Merge agahkarakuzu's changes upstream

### PR DESCRIPTION
**Pull request by joergbrech**
@agahkarakuzu, I like the changes from your moxunit action Fork. Do you mind if I merge them upstream?

**Changes:**
 - Use `qmrlab/octaveci:v4.2.2` instead of `mtmiller/octave` as base docker image
 - Add optional test data directory
 - Add option to install octave packages
 - Add option to add external packages to the search path

### CODE REVIEW

**Comment by agahkarakuzu** [comment]
@joergbrech by all means! 
> * Use `qmrlab/octaveci:v4.2.2` instead of `mtmiller/octave` as base docker image

If Octave `4.2.2` (probably) downgrade is not going to be a problem, feel free to merge this fork.

Example use case is available here:
https://github.com/agahkarakuzu/eda_organized/blob/master/.github/workflows/moxunit.yml

For a future release I can push a Docker image that contains a newer version of Octave with (more) packages installed. I probably won't these images as frequently as `mtmiller/octave` does, but packages are important to many people, probably a plausible tradeoff .

File: README.md [hunk 1]
```diff
@@ -41,6 +56,9 @@ steps:
 | - | - | - |
 | `tests` | files or directories containing the MOxUnit test cases | *Optional*, defaults to the root directory of the repo. All subdirectories are added recursively. |
 | `src` | directories to be added to the Octave search path before running the tests. These directories will be considered in the coverage reports, if coverage is enabled. | *Optional*
+| `data` | directory for test data | *Optional* |
+| `pkg` | Octave packages to load. Available options: `image` `io` `optim` `parallel` `statistics` `struct`| *Optional* |
+| `pkg` | External resources to add to the search put (excluded from coverage)| *Optional* |
```
Comment (joergbrech) [inline README.md:61]: I just found a typo: This should be `ext` and not `pkg`. I can add it myself, but maybe you want to fix this in your fork as well, @agahkarakuzu

**Comment by joergbrech** [comment]
Awesome, thanks! I don't think the downgrade is a problem IMHO. It probably makes sense to not use the newest release for unit testing anyway

**Comment by agahkarakuzu** [comment]
@joergbrech I fixed the typo, thanks :) Should I give a link to the https://github.com/agahkarakuzu/eda_organized/blob/master/.github/workflows/moxunit.yml repository in the examples?

**Comment by joergbrech** [comment]
> Should I give a link to the https://github.com/agahkarakuzu/eda_organized/blob/master/.github/workflows/moxunit.yml repository in the examples?

Oops, I was too fast. Sure, why not? 😄

**Pull request merged**